app/src/main/java/com/codepath/travel/adapters/viewholders/SuggestedPlaceViewHolder.java

```diff
@@ -37,9 +37,9 @@ public void populate(SuggestionPlace suggestionPlace) {
             rbPlaceRating.setRating(suggestionPlace.getRating().floatValue());
         }
         if (suggestionPlace.isSelected()) {
-            ivAddSuggestionPlace.setImageResource(android.R.drawable.star_big_on);
```

Comment:
Actually I was thinking of replacing the star in detail view with add/tick icon since it is more intuitive and then use checkbox. Will do that in next PR